src/mlagility/api/api.py

```diff
@@ -113,6 +113,7 @@ def benchit(
     build_name: Optional[str] = None,
     cache_dir: str = filesystem.DEFAULT_CACHE_DIR,
     device: str = "groq",
+    backend: str = "local",
```

Comment:
This should be `ip: str = "localhost"` (see user guide)

Reply:
`backend` seems like a more appropriate name here as we could either pass an IP address (after #6) or read from a config file as its done now. Can we consider editing the user guide to reflect this?

Reply:
Added https://github.com/groq/mlagility/issues/98